PerfNext/app/apis/BenchEngine/parser.js

```diff
@@ -376,12 +411,7 @@ function parseDefinition(raw_definition, callback){
 			generatedOutput += eval('`'+header+'`'); //Evaluate header params (${XXXXXX})
 			generatedOutput += '\n######### Generated Script #########\n';
 
-			if(iterations > 1){
-				generatedOutput += "iteration=0 \n";
-				generatedOutput += `while [ "$iteration" -lt ${iterations} ]` + '\n';
-				generatedOutput += 'do \n';
-				generatedOutput += "echo \"Start of iteration $iteration\" \n";
-			}
+			generatedOutput += `echo "iterations = ${iterations}"\n`;
```

Comment:
It's good to output this but maybe we can directly get the iterations value in `parseDefinition` instead of using this.